addons/web/static/tests/fields/relational_fields/field_one2many_tests.js

```diff
@@ -5776,8 +5775,8 @@ QUnit.module('fields', {}, function () {
 
                 "button_disabled_partner_4",
                 "button_warn_partner_4",
-                "button_disabled_partner_8",
-                "button_warn_partner_8"
+                "button_disabled_partner_5",
+                "button_warn_partner_5"
```

Comment:
why did this change?

Reply:
id auto increment was changed from doing the sum of all previous ids to doing last id + 1

Reply:
ok